src/seedu/addressbook/parser/Parser.java

```diff
@@ -50,7 +50,7 @@ public Parser() {}
      * @param userInput full user input string
      * @return the command based on the user input
      */
-    public Command parseCommand(String userInput) {
+    public static Command parseCommand(String userInput) {
```

Comment:
changing the methods static would usually require some other changes. were there any other changes to be made?